Functionality_Tests/BaseTest.cs

```diff
@@ -10,7 +10,7 @@
 
 namespace Functionality_Tests_Suit
 {
-    public abstract class BaseTest
+    public class BaseTest
     {
         public IWebDriver driver;
         public string mainUrl;
```

Comment:
You can use it in SeleniumTests and if so, it should be protected

Reply:
fixed